MapboxVisionTests/RecordingQuotaTests.swift

```diff
@@ -0,0 +1,124 @@
+import XCTest
+@testable import MapboxVision
+
+typealias Byte = Int64
+
+private let bytesInKByte: Byte = 1024
+private let kByteInMByte: Byte = 1024
+
+private let kByte = bytesInKByte
+private let mByte = kByteInMByte * kByte
+
+private let secondsInMinute: TimeInterval = 60
+private let minutesInHour: TimeInterval = 60
+
+private let minute = secondsInMinute
+private let hour = secondsInMinute * minute
+
+private let memoryLimit = 300.0 // mb
+private let networkingMemoryLimit = 30 * mByte
+private let updatingInterval = 1 * hour
+
+class RecordingQuotaTests: XCTestCase {
+    // MARK: - Properties
+
+    private var refreshInterval: TimeInterval!
+    private var initialMemoryQuota: Byte!
+    private var recordingQuota: RecordingQuota!
+
+    // MARK: - Test functions
+
+    override func setUp() {
+        super.setUp()
+        refreshInterval = TimeInterval.random(in: 5...10)
+        initialMemoryQuota = Byte.random(in: 5...10) * kByte
+
+        recordingQuota = RecordingQuota(memoryQuota: initialMemoryQuota, refreshInterval: refreshInterval)
+    }
+
+    override func tearDown() {
+        recordingQuota = nil
+        resetUserDefaultsState()
+        super.tearDown()
+    }
+
+    func testReserveMethodDoesNotThrowIfTriesToReserveZeroBytes() {
+        // Given
+        let memoryZeroBytes: Byte = 0
+
+        // When // Then
+        XCTAssertNoThrow(_ = try recordingQuota.reserve(memoryToReserve: memoryZeroBytes))
```

Comment:
Seems like we don't need `_ = ` before try, here and later.

Reply:
Agree.